pandas/tests/indexes/interval/test_setops.py

```diff
@@ -180,8 +180,8 @@ def test_set_incompatible_types(self, closed, op_name, sort):
         # GH 19016: incompatible dtypes
         other = interval_range(Timestamp("20180101"), periods=9, closed=closed)
         msg = (
-            "can only do {op} between two IntervalIndex objects that have "
-            "compatible dtypes"
-        ).format(op=op_name)
+            f"can only do {op_name} between two IntervalIndex objects that have "
+            f"compatible dtypes"
```

Comment:
There no need for an f-string in the second line

Reply:
Several of those, I included them to improve clarity. If the case is that we don't want them I can remove them. But I think they do not harm and improve clarity

Reply:
+/-0 on this, its fine